cmd/baseserver/README.md

```diff
@@ -24,11 +24,17 @@ Route | Method | Completed | Description
 /sign|POST|✔️|Sign a transaction
 /tx|POST|✖️|Post a transaction to the blockchain
 /seeds/status|GET|✖️|Returns the information on the last seed
+/build/create_role|POST|✔️|Creates a role. Please note that the role MUST be valid hex for example instead of sending "role", send its hex encoded equivalent "726f6c65"
```

Comment:
Not surprised to see this, but it doesn't make the best experience. Can we do something about it or is this a deep issue ?